gamemodes/horde/gamemode/sh_rank.lua

```diff
@@ -113,7 +113,7 @@ function plymeta:Horde_SyncExp()
 end
 
 function HORDE:GetExpToNextLevel(level)
-    return math.floor(50 + 2 * math.pow(1.1, level) + level * 25 + 100 * math.floor(level / 5))
+    return math.floor((50 + 2 * math.pow(1.1, level) + level * 25 + 100 * math.floor(level / 5))*10)
```

Comment:
would make more sense to be ~5 or so, as xp will be significiantly harder to get on all difficulties

Reply:
i'll set it to 6